frontend/sw360-portlet/src/main/java/org/eclipse/sw360/portal/common/PortletUtils.java

```diff
@@ -75,6 +76,16 @@ public class PortletUtils {
     private static final String TEMPLATE_FILE = "/welcomePageGuideline.html";
     private static final ObjectMapper objectMapper = new ObjectMapper();
     private static ChangeLogsPortletUtils changeLogsPortletUtils = null;
+    private static String EXTERNALID_BLOCKLIST_CLONING_RELEASE;
+    private static final String PROPERTIES_FILE_PATH = "/sw360.properties";
+    
+    static {
+        CommonUtils.loadProperties(PortletUtils.class, PROPERTIES_FILE_PATH);
+        Properties props = CommonUtils.loadProperties(PortletUtils.class, PROPERTIES_FILE_PATH);
+
+        EXTERNALID_BLOCKLIST_CLONING_RELEASE = props.getProperty("externalId.remove.while.cloning.release", "");
```

Comment:
May be rename it to `release.externalId.to.be.removed.while.cloning` for better readability.